cli/src/main.rs

```diff
@@ -17,27 +17,7 @@ fn to_commit_hash(s: &str) -> Result<CommitHash> {
     Ok(CommitHash { hash })
 }
 
-#[tokio::main(flavor = "multi_thread")]
-#[allow(unreachable_code)]
-async fn main() -> eyre::Result<()> {
-    color_eyre::install().unwrap();
-    env_logger::init();
-
-    let private_key = std::env::args().nth(1).unwrap();
-    let sorc = std::env::args().nth(2).unwrap();
-    if sorc == "s" {
-        genesis::run_genesis_proposer(&private_key).await;
-    } else {
-        genesis::run_genesis_non_proposer(&private_key).await;
-    }
-
-    return Ok(());
-
-    let args = cli::Cli::parse();
-    let path = args.path.display().to_string();
-    let config: Config =
-        serde_spb::from_str(&tokio::fs::read_to_string(&format!("{}/config.json", path)).await?)?;
-
+async fn run(args: cli::Cli, path: String, config: Config) -> eyre::Result<()> {
```

Comment:
What is the path for?

Reply:
see the line 28.

Reply:
So, it's a path to the repository?

Reply:
path of the root directory of a Simperby blockchain.  See `directory.md`